ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+
+### 2.1.0
```

Comment:
Sorry, by PVP that's a major version bump. The next minor version would be 2.0.19.

Reply:
updated